documentation/docs/05-misc/02-accessibility-warnings.md

```diff
@@ -52,16 +52,18 @@ Enforce that `autofocus` is not used on elements. Autofocusing elements can caus
 
 ## `a11y-click-events-have-key-events`
 
-Enforce `on:click` is accompanied by at least one of the following: `on:keyup`, `on:keydown`, `on:keypress`. Coding for the keyboard is important for users with physical disabilities who cannot use a mouse, AT compatibility, and screenreader users.
+Enforce that visible, non-interactive elements with an `on:click` event are accompanied by a keyboard event handler.
 
-This does not apply for interactive or hidden elements.
+Users should first consider whether an interactive element might be more appropriate such as a `<button type="button">` element for actions or `<a>` element for navigations. These elements are more semantically meaningful and will have built-in key handling. E.g. `Space` and `Enter` will trigger a `<button>` element's navigation while `Space` will focus an `<a>` element and `Enter` will trigger an `<a>` element's navigation.
```

Comment:
oh, yeah, good catch - that was a typo. I'd meant to write "action" rather than "navigation"
```suggestion
Users should first consider whether an interactive element might be more appropriate such as a `<button type="button">` element for actions or `<a>` element for navigations. These elements are more semantically meaningful and will have built-in key handling. E.g. `Space` and `Enter` will trigger a `<button>` element's action while `Space` will scroll to an `<a>` element and `Enter` will trigger an `<a>` element's navigation.
```

Reply:
Thanks, but that is still not quite right.
Space just scrolls the page down, it does not regard the link in any way, which is why I suggested to leave that out.

Reply:
ah, thanks for clarifying. updated!